packages/eslint-config/package.json

```diff
@@ -14,6 +14,9 @@
     "eslint-config-prettier": "8.3.0",
     "eslint-config-react-app": "7.0.0"
   },
+  "devDependencies": {
+    "eslint-plugin-deephaven": "file:./plugin"
+  },
```

Comment:
Looks like you don't need to do it this way. You can import the plugin directly

https://eslint.org/docs/latest/extend/custom-rule-tutorial#step-7-bundle-the-custom-rule-in-a-plugin

Reply:
This doesn't work if you try to add it to an array of plugins since the plugins array only supports string names. Granted the current config only has 1 plugin, this will be more future proof.

That said if you find a way to get it working alongside other plugins, I'd love to see how. I was not able to get it working.

Reply:
@mattrunyon @mofojed Thinking this PR may be a cleaner option
https://github.com/deephaven/web-client-ui/pull/1499/files